packages/cli/package/src/lib/lifeCycle.ts

```diff
@@ -115,22 +87,16 @@ export async function initCli<
     );
   }
 
-  await ensureUserConfig();
+  await initNewUserConfig();
 
   if (requiresFluenceProject) {
-    setEnvConfig(await initNewEnvConfig());
+    await initNewEnvConfig();
   } else {
     try {
       // ensure env config also when provider.yaml exists
       await access(getProviderConfigPath());
-      setEnvConfig(await initNewEnvConfig());
-    } catch {
-      const envConfig = await initEnvConfig();
```

Comment:
We don't need that anymore?

Reply:
I added a global "cache" for all configs long ago, cause essentially when you init a config object you can reuse it for the duration of CLI running cause it is expected to move anywhere, so there is no need anymore to store it globally separately. Finally removed that